diff --git a/wrangler.local.jsonc b/wrangler.local.jsonc
index 7ea8359..db56895 100644
--- a/wrangler.local.jsonc
+++ b/wrangler.local.jsonc
@@ -4,8 +4,8 @@
   "d1_databases": [
     {
       "binding": "DB",
-      "database_name": "site-creator-d1",
-      "database_id": "00000000-0000-4000-8000-000000000000",
+      "database_name": "food-tracker-db",
+      "database_id": "e998b960-0351-4cfa-8043-41780649bf13",
     },
   ],
 }
